python/lsst/ts/xml/data/sal_interfaces/TunableLaser/TunableLaser_Events.xml

```diff
@@ -34,6 +34,18 @@ LaserErrorCode_HW_CPU_ERROR = 7304
       <Count>1</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>TunableLaser</Subsystem>
+    <EFDB_Topic>TunableLaser_logevent_alignmentChanged</EFDB_Topic>
```

Comment:
This name gives the impression of a boolean (did the alignment changed or not?). Maybe replace this with, `opticalConfiguration`?